File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinWarningDto.java
package org.jdk.project.dto.digitaltwin;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinWarningDto {
  private Long id;
  private String warningType;
  private String warningInfo;
  private String noticeType;
  private String resultStatus;
  private String isSuccess;
  private String createdAt;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinWarningDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinWarningDto(
    var id: Long? = null,
    var warningType: String? = null,
    var warningInfo: String? = null,
    var noticeType: String? = null,
    var resultStatus: String? = null,
    var isSuccess: String? = null,
    var createdAt: String? = null,
)
